pydoctor/test/test_astbuilder.py

```diff
@@ -582,6 +604,8 @@ class C:
         @ivar e: fifth
 
         @type f: C{str}
+
+        @type g: C{str}
```

Comment:
In #190 you replaced `C{str}` with `string` FWIW.

Reply:
Well spotted. This has to be changed to `string` as well; I'm doing so in the rebase.